CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 - Make pg_repack the default way to perform maintenance [#25](https://github.com/npwalker/pe_databases/pull/25)
 - Start maintaining the reports table which we could not afford to perform a VACUUM FULL on
-- Rename typoed `pe_databases::maintenance::disable_maintenace` parameter to `pe_databases::maintenance::disable_maintenance`
+- Rename typoed `pe_databases::maintenance::disable_maintenance` parameter to `pe_databases::maintenance::disable_maintenance`
```

Comment:
ummm you fixed the typo but the typo needs to stay

Reply:
Barf